modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -7,59 +7,29 @@
 
 You can establish a binding connection with Operator-backed components.
 
-This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Spring PetClinic sample application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you must first add the Red Hat-provided PostgreSQL Database Operator to the *OperatorHub*, and then install the Operator.
-The PostreSQL Database Operator then creates and manages the `Database` resource, which exposes the binding information in secrets, config maps, status, and spec attributes.
+This procedure walks through an example of creating a binding connection between a PostgresSQL Database service and a Spring PetClinic sample application.
+
+To create a binding connection with a service that is backed by the Crunchy Postgres for Kubernetes, you must add the Crunchy Postgres for Kubernetes operator to the *OperatorHub*, and then install the Operator. The Crunchy Postgres for Kubernetes Operator provides PostgresCluster resource which then can be used to deploy a  PostgreSQL database, which exposes the binding information in secrets, config maps, status, and spec attributes.
+
+When you create the service binding, the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
 
 .Prerequisites
 
 * You have created and deployed a Spring PetClinic sample application by using the *Developer* perspective.
 * You have installed {servicebinding-title} from the OperatorHub.
```

Comment:
In contrast, `the *Developer* perspective` almost always uses bold formatting because it exists purely as a UI element in the web console. For example:
```
If you want to add or edit health checks in an existing pod, you must edit the pod `DeploymentConfig` object or use the *Developer* perspective in the web console. You cannot use the CLI to add or edit health checks for an existing pod.
```

Reply:
Good to know, Rolfe. My experience with OperatorHub has been limited to the OpenShift web console, where it is indeed a UI label, **OperatorHub**, so in my comment above, I followed the style guide guidance from https://redhat-documentation.github.io/supplementary-style-guide/#user-interface-elements, "Bold all graphical user interface elements, including, but not limited to, menu names, menu items, button names ..."

Reply:
Fixed.